src/decapodes_it.jl

```diff
@@ -0,0 +1,272 @@
+module ASKEMDecapodes
+
+export ASKEMDecaExpr, ASKEMDecapode, ASKEMDeca, SummationDecapode, parse_decapode
+
+# using ..SyntacticModelsBase
+using ..AMR
+
+using StructTypes
+# using Decapodes
```

Comment:
is anything using Decapodes now that you copied code over here? Should we upgrade Decapodes to use InterTypes and then just use InterType inteheritance here?